src/renderer/views/multiplayer/custom.vue

```diff
@@ -4,7 +4,8 @@
 
 <template>
     <div class="flex-row flex-grow gap-md hide-overflow">
-        <div class="flex-col flex-grow gap-md">
+        <Loader v-if="loading"></Loader>
+        <div v-if="!loading" class="flex-col flex-grow gap-md">
```

Comment:
You can replace `v-if="!loading"` here with `v-else`

Reply:
changed